src/CatapultECC.ts

```diff
@@ -18,7 +18,9 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-import {array, nacl_catapult, sha3Hasher} from 'nem2-library';
+
+import * as nacl_catapult from 'nem2-sdk/dist/src/core/crypto/nacl_catapult';
```

Comment:
you don't need to put the `dist` path in there.

Just put `nem2-sdk` so that it becomes:

`import { nacl_catapult } from 'nem2-sdk';`